### PR TITLE
 LPS-86596 Add Custom Filter Widget

### PR DESCRIPTION
**Pull request by luanmaoski**

### CODE REVIEW

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/custom/relevance/CustomRelevanceContainer.java [hunk 1]
```diff
@@ -45,6 +46,22 @@ public void addBoosterTerms(BooleanQuery booleanQuery) {
 		}
 	}
 
+	@SuppressWarnings("unchecked")
+	public void addCustomBoosterTerms(
+		ArrayList<Object[]> customBoostList, BooleanQuery booleanQuery) {
+
+		if (customBoostList != null) {
+			for (Object[] customBoost : customBoostList) {
+				String field = (String)customBoost[0];
```
Comment (luanmaoski) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/custom/relevance/CustomRelevanceContainer.java]: @brandizzi an we move the class "CustomRelevance" to the "portal-search-api" module to use it here?

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/custom/relevance/CustomRelevanceContainer.java (hunk 1 above)
Comment (brandizzi) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/custom/relevance/CustomRelevanceContainer.java]: @luanmaoski yes, that makes sense!

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brandizzi** [comment]
Merged into https://github.com/brandizzi/liferay-portal/commits/BOOSTING. I've made some small changes to resolve conflicts and make some formatting. Thanks!

**Pull request closed without merging**